src/content/1.7/modules/creation/module-translation/new-system.md

```diff
@@ -140,11 +140,11 @@ Since the module is called MyModule, the translation domain should be `Modules.M
 ```php
 // file: controllers/front/something.php
 
-class MyModuleSomethingFrontController extends ModuleFrontController
+class MySomethingModuleFrontController extends ModuleFrontController
```

Comment:
This is wrong. If the module is called `mymodule` then the controller `Something` must be called like this:

```suggestion
class MymoduleSomethingModuleFrontController extends ModuleFrontController
```

> the class name follows this format: `<ModuleName><FileName>ModuleFrontController`.

[source](https://devdocs.prestashop.com/1.7/modules/concepts/controllers/front-controllers/)

Reply:
Indeed, bad copy-paste